addons/account/views/res_config_settings_views.xml

```diff
@@ -47,11 +47,11 @@
                                 <div class="content-group">
                                     <div class="row mt16">
                                         <label string="Sales Tax" for="sale_tax_id" class="col-lg-3 o_light_label"/>
-                                        <field name="sale_tax_id" widget="many2one_tax_tags" domain="[('type_tax_use', 'in', ('sale', 'all'))]"/>
```

Comment:
This will log a warning in the browser because the widget doesn't exist anymore, but nothing more as far as I can tell.
Not a (big) issue for stable